### ISSUE TITLE
Would like a MultilineConditionalLayout cop

### ISSUE THREAD

**Issue by tleish** [rubocop-hq/rubocop#7392]
## Is your feature request related to a problem? Please describe.

Single line conditionals have some pros and cons.

Pros:
- Single line conditionals allow me to pack more code into less lines (Ruby golf)
- Postfix conditionals can often read well (`execute unless false`).

Cons:
- When analyzing with code coverage tools, single line conditionals mask code coverage.  Code coverage will always show as true for that line of code, even if the internals of the conditional are not exercised.
- Can sometimes decrease readability of the code.
- It can be difficult to quickly see code branching logic when scanning lines of code to finding a possible cause of a bug,

## Describe the solution you'd like

I'd like a cop that allows me to enforce conditional layout styles
Style/MultilineConditionalLayout
```ruby
# bad
execute if x < y

# good
if x > y
  execute
end
```

```ruby
# bad
puts x < y ? 'less than' : 'greater than or equal to'

# good
if x < y
  puts 'less than'
else
  puts 'greater than or equal to'
end

# good
puts if x < y
  'less than'
else
  'greater than or equal to'
end
```

## Describe alternatives you've considered
Ruby 2.5 now supports Coverage branching (see https://bugs.ruby-lang.org/issues/13901).  It has not yet been implemented in the Simplecov gem (see: https://github.com/colszowka/simplecov/pull/694).

Alternatively you could use the [DeepCover](https://github.com/deep-cover/deep-cover) gem to analyze code coverage branching.

While this would solve the code coverage concern, it does not solve the readability scanning for branching.

## Other comments
- Not sure if this be considered a Style or Layout cop?
